drivers/soundwire/sysfs_slave.c

```diff
@@ -212,3 +216,56 @@ int sdw_slave_sysfs_init(struct sdw_slave *slave)
 
 	return 0;
 }
+
+static const char *const slave_status[] = {
+	[SDW_SLAVE_UNATTACHED] =  "UNATTACHED",
+	[SDW_SLAVE_ATTACHED] = "Attached",
+	[SDW_SLAVE_ALERT] = "Alert",
+	[SDW_SLAVE_RESERVED] = "Reserved",
```

Comment:
mmh, sorry, but this is a part of the user API so, seems to be important to get it right from the beginning not to break any scripts by changing it later. Is this really intended like this - "UNATTACHED" all capitalised and all other status words with the first capital letter only?

Reply:
That was @vinodkoul's code, I took it as is.
I guess it makes sense to use capitals to show this isn't a nominal condition, either the device is not physically present or it went in the weeds and is no longer detected as attached on the bus.